app/core/Tracker2/index.js

```diff
@@ -40,12 +40,23 @@ export default class Tracker2 extends BaseTracker {
     this.trackers = [
       this.legacyTracker,
       // this.googleAnalyticsTracker,
-      this.driftTracker,
       this.segmentTracker,
       this.fullStoryTracker,
-      this.googleOptimizeTracker,
-      this.facebookPixelTracker
     ]
+
+    const isGlobal = !(window.features || {}).china
+    if (isGlobal) {
+      // add trackers we don't want china to enable here.
+       var globalTrackers = [
```

Comment:
This could be set directly on line line 57, skipping this variable completely.

Reply:
Okay